src/store/actions/cohorts.actions.js

```diff
@@ -13,3 +13,22 @@ export const fetchCohortsThunk = () => dispatch => {
     .get(API_URL)
     .then(res => dispatch(setCohortsData(res.data.data)));
 };
+
+export const fetchActiveCohort = cohortID => dispatch => {
```

Comment:
As discussed on our call, I think this thunk function to get the current active cohort should be taken out into a separate action file called something like singleCohort.actions.js